.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: You have an idea on how Tatoeba could be improved
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+*Please tell us a story.* How did you came up with that idea? What made you think about it?
+
+*Example:*
+> On the Wall, I often find some discussions very interesting, but I don't feel like commenting because I have nothing to say. One recent example is [this one](https://tatoeba.org/eng/sentences/show/8619497).
+> But I happen upon something really interesting like this quite often. Sometimes I stumble upon an interesting question that is not answered yet, and I want to wait for the answer too. I just don't want to miss it, and I believe I mostly miss them.
+> For questions, especially if the question has been there for years, I sometimes add something like "I've got this question too" - it's not really constructive, but this way it gets a higher chance of being noticed, and it also "subscribes" me to that thread.
+> But I often don't have anything to contribute, but still interested in the discussion and don't want to miss anything, taking into account some discussions are happening at the speed of one post per month or something.
+
+Finally, please *focus on the problem* that your feature request is solving. What’s wrong with how things are now? Wrap it up into one sentence and use it as title for the issue.
+*Example:* Can't subscribe to a discussion without posting an unnecessary comment
```

Comment:
This last part could be moved to the top of the template because it gives tips about the title and that's probably what people will write first when they create an issue.

Could be rephrase this way:

> For the title, summarize in one sentence what is the problem that your feature request is solving.
> *Example: Can't subscribe to a discussion without posting an unnecessary comment*